pytorch_lightning/trainer/supporters.py

```diff
@@ -162,4 +172,5 @@ def to_disk(self):
                 outputs.append(output_element)
 
             # Write predictions for current file to disk
-            torch.save(outputs, outfile)
+            with fs.open(filepath, "wb") as fp:
```

Comment:
how about back-compatibility for loading also with "wb"

Reply:
since it's saved with torch.save, you'd need to use torch.load to read, which uses mode="rb": https://pytorch.org/docs/stable/_modules/torch/serialization.html#load